### ISSUE TITLE
"is " translated into "is not " in test output

### ISSUE THREAD

**Issue by hubbyist** [Codeception/Codeception#3801]
#### What are you trying to achieve?
Trying to see response which echos "is OK" phrase with $I->dontSeeResponseContains("ERROR") statement.

#### What do you get instead?
Test works but console test report shows "is OK" translated to "is not OK".

### Details
Only 'is OK' translated to 'is not OK'. 'isOK' or 'is-OK' is untouched.

* Codeception version: Codeception version 2.2.6
* PHP Version: PHP 7.0.8-0ubuntu0.16.04.3
* Operating System: linux mint 18.0
* Installation type: Phar
* List of installed packages (composer show) : none
* Suite configuration:

```yml
class_name: AcceptanceTester
modules:
    enabled:
        - Asserts
        - PhpBrowser:
            url: http://example.com/sitefolder/folder
        - \Module\RESTAPI:
            depends: PhpBrowser
        - Db:
            dsn: 'pgsql:host=localhost;dbname=dbname'
            user: 'username'
            password: 'password'
            dump: dump.sql
            populate: true
            cleanup: false
            reconnect: false
```

**Comment by hubbyist** [comment]
"are " becoming "are not" as well.

**Comment by DavertMik** [comment]
maybe because `dontSeeResponseContains` negates the response message?
could you show actual console output?

**Comment by hubbyist** [comment]
It seems so. When I run ``` codecept -vvv run acceptance ```, output is negated after ```I don't see response contains "ERROR"``` line.

console: output
```
  [Request Cookies] {"PHPSESSID":"mt6841veui7o7fts4bklopu6i6"}
  [Response Headers] {"Date":["Mon, 05 Dec 2016 06:40:09 GMT"],"Server":["Apache/2.4.18 (Ubuntu)"],
"Expires":["Thu, 19 Nov 1981 08:52:00 GMT"],"Cache-Control":["no-store, no-cache, must-revalidate"],
"Pragma":["no-cache"],"Content-Length":["719"],"Content-Type":["text/html; charset=utf-8"]}
  [Response] 
  1.    : identifier is numeric
  1=    : identifier is numeric
  2.    : identifier is numeric
  2=    : identifier is numeric
  3.    : identifier is numeric
  3=    : identifier is numeric
  4.    : identifier is numeric
  4=    : identifier is numeric
  5.    : identifier is numeric
  6.    : identifier is numeric
  7.    : identifier is numeric
  5=    : identifier is numeric
  6=    : identifier is numeric
  7=    : identifier is numeric
  8. : identifier is numeric
  8=    : identifier is numeric
  9. : identifier is numeric
  9=    : identifier is numeric
  10. : identifier is numeric
  10=   : identifier is numeric
  11. : identifier is numeric
  11=   : identifier is numeric
  12.   : identifier is numeric
  12=   : identifier is numericNULLERROR
 I don't see response contains "ERROR"
 FAIL 

----------------------------------------------------------------------------------------------------


Time: 2.01 seconds, Memory: 16.00MB

There was 1 failure:

---------
1) ManageImportCest: Import exam
 Test  tests/acceptance/ManageImportCest.php:ImportExam
 Step  Don't see response contains "ERROR"
 Fail  REST response contains
Failed asserting that '
1.    : identifier is not numeric
1=    : identifier is not numeric
2.    : identifier is not numeric
2=    : identifier is not numeric
3.    : identifier is not numeric
3=    : identifier is not numeric
4.    : identifier is not numeric
4=    : identifier is not numeric
5.    : identifier is not numeric
6.    : identifier is not numeric
7.    : identifier is not numeric
5=    : identifier is not numeric
6=    : identifier is not numeric
7=    : identifier is not numeric
8. : identifier is not numeric
8=    : identifier is not numeric
9. : identifier is not numeric
9=    : identifier is not numeric
10. : identifier is not numeric
10=   : identifier is not numeric
11. : identifier is not numeric
11=   : identifier is not numeric
12.   : identifier is not numeric
12=   : identifier is not numericNULLERROR' does not contain "ERROR".

Scenario Steps:

 19. $I->dontSeeResponseContains("ERROR") at tests/acceptance/ManageImportCest.php:38
```

**Comment by Naktibalda** [comment]
It is caused by a bug of PhpUnit - https://github.com/sebastianbergmann/phpunit/issues/2098

**Comment by Naktibalda** [comment]
This issue was fixed in PHPUnit 6.4.2
